CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# v2.3.2 (2024-08-13)
+
+## OS Changes
+
+* Kernel 6.1.102-108.177 ([#73])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Update ecr-credential-provider to latest upstream ([#66])
+* Update soci-snapshotter to latest upstream (vended in aws-dev variant) ([#66])
```

Comment:
kind of nit picking and slightly contradicting my earlier comment but I think this could go either more or less detailed than you have. More:

```suggestion
* ecr-credential-provider: update to 1.25.15 ([#66])
* ecr-credential-provider-1.27: update to 1.27.8 ([#66])
* ecr-credential-provider-1.29: update to 1.29.6 ([#66])
* ecr-credential-provider-1.30: update to 1.30.3 ([#66])
* soci-snapshotter: update to 0.7.0 ([#66])
```

Less (and closer to how it was expressed in Bottlerocket variant release changelogs):

```suggestion
* Update third party packages ([#66])
```

My personal preference is the more detailed listing.

Reply:
My personal preference is also the more detailed listing, thanks for the suggestion.